pallets/capacity/src/lib.rs

```diff
@@ -195,6 +195,15 @@ pub mod pallet {
 	#[pallet::getter(fn get_current_epoch_used_capacity)]
 	pub type CurrentEpochUsedCapacity<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
 
+	#[pallet::type_value]
+	/// EpochLength defaults to T::MaxEpochLength when not set
```

Comment:
Let's consider dropping Max from MaxEpochLengh.

Reply:
Also, can we remove the configuration if we now have a default value and are setting it through governance?

Reply:
I think the intention for `MaxEpochLength` was to set an upper bound for how high we want to allow the epoch length to go, but I suppose if it is up to governance, we can just use a default

Reply:
After discussion internally, we decided to keep `MaxEpochLength` in place for safety